src/cockpit/389-console/package.json

```diff
@@ -10,47 +11,41 @@
   "license": "GPL-3.0-or-later",
   "main": "index.js",
   "scripts": {
-    "build": "webpack",
-    "watch": "webpack --watch --progress",
-    "audit-ci": "audit-ci --config audit-ci.json",
```

Comment:
When I try to run audit-ci, it asks me for the dependency.

```
npx audit-ci
Need to install the following packages:
  audit-ci
Ok to proceed? (y) y
```

I think we should add it back to src/cockpit/389-console/package.json...

Reply:
> When I try to run audit-ci, it asks me for the dependency.
> 
> ```
> npx audit-ci
> Need to install the following packages:
>   audit-ci
> Ok to proceed? (y) y
> ```
> 
> I think we should add it back to src/cockpit/389-console/package.json...

Done